Jellyfin.Api/Controllers/SystemController.cs

```diff
@@ -71,6 +71,19 @@ public SystemController(
     public ActionResult<SystemInfo> GetSystemInfo()
         => _systemManager.GetSystemInfo(Request);
 
+    /// <summary>
+    /// Gets information about the server.
+    /// </summary>
+    /// <response code="200">Information retrieved.</response>
+    /// <response code="403">User does not have permission to retrieve information.</response>
+    /// <returns>A <see cref="SystemInfo"/> with info about the system.</returns>
+    [HttpGet("StorageResources")]
+    [Authorize(Policy = Policies.FirstTimeSetupOrElevated)]
```

Comment:
What is the reasoning for allowing first time setup to access this endpoint?

Reply:
The setup UI could also benefit from displaying this information as it also is used to add libraries

Reply:
Let's keep it admin only for now, we can always enable it for setup once jf-web is going to use it.